challenges/odd-divisor/README.md

```diff
@@ -0,0 +1,18 @@
+# Challenge: Odd Divisor
+
+Goal:
+Define function f as the solution to get the largest odd divisor of x, x is a positive integer, e.g. f(44)=11,
+now given n, calculate the value of f(1) +  f(2)  + f(3) + ... + f(n).
+For example:
+n=7
+f(1) + f(2) + f(3) + f(4) + f(5) + f(6) + f(7) = 1 + 1 + 3 + 1 + 5 + 3 + 7 = 21
+input: 1 ≤ n ≤ 1000000000, or even larger    
+
+The fastest function wins. Good luck!
+
+## Requirements
+
+- Fastest performing solution wins
+- Must work in browser and node.js
```

Comment:
Maybe remove this...
Any language should be allowed for this challenge.

Reply:
Actually not.
Javascript is not good enough for this, I will explain that later in the lunch n learn.